webpack.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
just curious why we need webpack config  @anshitmishra

Reply:
webpack will handle module CSS. It allows us to import CSS on a single page @Yashsharma1911

Reply:
thanks for confirming sounds real good